js/filter.js

```diff
@@ -156,14 +161,14 @@ const _buildKeywordQuery = (field, operator, left, right) => {
     switch (operator) {
         case "gt":
         case "gte":
-            return [`${field}:${operator}:${right.getTime() / 1000}`];
+            return [`${field}:${operator}:${parseInt(right.getTime() / 1000)}`];

```

Comment:
`Math.floor` seems more correct, as `parseInt` is usually used to convert strings into numbers.